### DISCUSSION TITLE
Azure DNS on Linux Web App

### DISCUSSION THREAD

**Discussion by asapostolov** [shibayan/appservice-acmebot#383] [Q&A] [answered]
I'm reading [here](https://github.com/shibayan/appservice-acmebot/issues/31) (and also the readme) that there is a hard requirement that the domain is managed through Azure DNS when setting up a Linux Web App. Is there any way to avoid this? For example if I set the `_acme-challange` TXT record manually in my other domain registrar?

I'm not in control of the domain which I'm setting up and moving it would be kind of a big deal.

**Comment by shibayan** [comment, marked as answer]
This can be solved by using the Key Vault version. The App Service version does not support anything other than Azure DNS.
https://github.com/shibayan/keyvault-acmebot

**Reply by asapostolov** [reply]
In that case the domain provider for DNS should be one of the ones [specified in the documentation](https://github.com/shibayan/keyvault-acmebot/wiki/DNS-Provider-Configuration#custom-dns-preview) that the Key Vault supports, right?

**Reply by shibayan** [reply]
Yes. You're right.
